pythonforandroid/toolchain.py

```diff
@@ -528,7 +537,9 @@ def add_parser(subparsers, *args, **kwargs):
         self.ndk_dir = args.ndk_dir
         self.android_api = args.android_api
         self.ndk_version = args.ndk_version
-        self.ndk_api = args.ndk_api
+        args.ndk_api = select_ndk_api(args.ndk_api, args.android_api)
+        self.ndk_api = args.ndk_Api
```

Comment:
Typo here ("ndk_api")

Reply:
Travis didn't like that either :smile: 
```
AttributeError: 'Namespace' object has no attribute 'ndk_Api'
```

Reply:
I have also tried, but got this error:

    args.ndk_api = select_ndk_api(args.ndk_api, args.android_api)
    NameError: name 'select_ndk_api' is not defined